security/pfSense-pkg-nmap/Makefile

```diff
@@ -34,6 +34,8 @@ do-install:
 		${STAGEDIR}${PREFIX}/pkg
 	${INSTALL_DATA} ${FILESDIR}/etc/inc/priv/nmap.priv.inc \
 		${STAGEDIR}/etc/inc/priv
+    ${INSTALL_DATA} ${FILESDIR}${PREFIX}/www/nmap_view_results.php \
+        ${STAGEDIR}${PREFIX}/www
```

Comment:
These lines have leading spaces but must be tabs, otherwise make fails.

Reply:
Sorry, I forgot nano was set to spaces instead of tabs, should have used ee
I usually work in VS, but the repo clone fails because of japanese/prn under windows:
https://github.com/freebsd/freebsd-ports/pull/122